ext/random/php_random.h

```diff
@@ -37,12 +37,19 @@
 
 PHPAPI double php_combined_lcg(void);
 
+static inline zend_long GENERATE_SEED()
```

Comment:
I think this should be lowercase, the general convention AFAIK is that uppercase means macro, lowercase means functions, especially for a private API

Reply:
Changing the casing would result in an unnecessary API break. As the function is `static inline` there is no behavioral change compared to the macro. Specifically it will not result in any more naming collisions than the macro would, but it does improve type safety and readability somewhat. It also allows to more easily adjust the seed generation, which is terrible (luckily `GENERATE_SEED()` is effectively only used if the CSPRNG fails, which it never does).

Reply:
Ah it's a static inline in a header.... yeah sure that's fine